src/position.rs

```diff
@@ -366,17 +365,15 @@ impl PositionManager {
             parameters.insert("symbol".into(), v.into());
         }
 
-        if let Some(start_str) = req.start_time.as_ref().map(|s| s.as_ref()) {
-            let start_millis = date_to_milliseconds(start_str);
+        if let Some(start_time) = req.start_time {
             parameters
-                .entry("end".to_owned())
```

Comment:
JSON key of `start_time` incorrectly called `"end"` - should have been `"startTime"`